docs/en/ingest-management/elastic-agent/elastic-agent-encryption.asciidoc

```diff
@@ -0,0 +1,29 @@
+[elastic-agent-encryption]
+= Elastic Agent Configuration Encryption
+
+It is important for users to understand the security model of Elastic Agent and how it handles sensitive values in integration configurations.
+At a high level, Elastic Agent receives configuration data from Fleet Server over an encrypted connection, persists those secrets in an encrypted form on disk.
+This persistence is necessary to allow agents to continue to operate, even in the event of a loss of connectivity to the Fleet server.
+
+The entirety of the Fleet Agent Policy is encrypted at rest, but is recoverable if the user has access to both the encrypted configuration data and the associated key.
+The key material in stored in an OS-dependent manner as described below.
+
+[discreet]
+== Darwin (Mac OS)
+
+Key material is stored in the system keychain. The value is stored as is without any additional transformations.
+
+[discreet]
```

Comment:
```suggestion
[discrete]
```

Reply:
IMO asciidoc's naming here is unfortunate. As it is, you end up with:

![image](https://user-images.githubusercontent.com/14206422/202062678-8a098e05-0600-40c7-80d0-b56c07684a68.png)